### ISSUE TITLE
Why use tensorflow1.3.0GPU training to get the validation loss is much higher than the CPU training

### ISSUE THREAD

**Issue by pocheyeniu** [tensorflow/tensorflow#13328]
when I use tf1.3.0 CPU vesion, the validation loss is 0.54; but get 0.74 using GPU instead. I can't handle it.
Cudnn version is 6.0

**Comment by aselle** [comment]
I apologize but I am having a hard time understanding what the problem is, where the problem is, and what version it affects. Please resubmit and pay attention to the issue template (https://github.com/tensorflow/tensorflow/issues/new) . Please provide all the information it asks. We test our code before releases, so this is possibly caused by your specific case. Open a new issue with a reproducible test case if you would like more help. Thank you.
